.evergreen/static-check.sh

```diff
@@ -11,4 +11,5 @@ source java-config.sh
 echo "mongo-hibernate: static checking ..."
 
 ./gradlew -version
-./gradlew -PxmlReports.enabled=true --info -x test clean check compileJava
+
+./gradlew --info -x test clean check compileJava
```

Comment:
I removed it for after testing I found regardless of whether we configure it, no xml report will be generated at all.
I guess the reason it works in Java driver is due to the usage of SpotBug plugin; but we are using `Spotless/Palantir` + `ErrorProne/NullAway`, so that configuration won't take effect.

Reply:
I dig a little bit. Seems for now both `Spotless` and `ErrorProne` won't generate any parsable report:

- https://github.com/diffplug/spotless/issues/811
- https://github.com/google/error-prone/issues/444

It might be worth while to keep that in mind if we plan to replace SpotBug with ErrorProne in Java driver.

Reply:
Can always add later.